packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -1272,6 +1295,25 @@ class ApiGateway {
     return [queryType, normalizedQueries, queryNormalizationResult.map((it) => remapToQueryAdapterFormat(it.normalizedQuery))];
   }
 
+  public async sql4sql({
```

Comment:
Should this be private?

Reply:
TBH, I'm not sure.
I just copy-pasted `public async sql({`, but I see no reason to keep this public